### ISSUE TITLE
Expansion Panel Content being swallowed

### ISSUE THREAD

**Issue by jt-helsinki** [angular/material2#7008]
#### Bug, feature request, or proposal:

Bug

#### What is the expected behavior?

The content of the panel will spill over the right edge of the panel when the screen resolution is too small.

**OR**

Content is scrollable.

#### What is the current behavior?

The panel swallows the content with no horizontal scroll bars. This hides information with no way to retrieve it. 

#### What are the steps to reproduce?

Simply put a table with a 1200px width inside an expansion panel. Shrink the screen horizontally.

See this plunk:

http://plnkr.co/edit/t0qEtVdYlJ6MFCNhVddo?p=preview

#### What is the use-case or motivation for changing an existing behavior?

Placing a table inside an expansion panel. 

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

Material 2.0.0-beta.10
Angular 4.3.6

**Comment by josephperrott** [comment]
Closed with #7012
